lib/utilities.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+from dateutil.parser import parse
+import pytz
+
+def toUTC(suspectedDate, localTimeZone='UTC'):
```

Comment:
Although, now that I think about #1, do we want to have the timezone be dynamic? Maybe we should make a hard rule that any timestamps in mozdef, will always try and be UTC?